core/src/main/scala/kafka/server/MetadataCache.scala

```diff
@@ -57,16 +58,20 @@ class MetadataCache(brokerId: Int) extends Logging {
   // we should be careful about adding additional logic here. Relatedly, `brokers` is
   // `Iterable[Integer]` instead of `Iterable[Int]` to avoid a collection copy.
   // filterUnavailableEndpoints exists to support v0 MetadataResponses
```

Comment:
Part of this comment doesn't seem accurate anymore. For example, `brokers` is now a `List`?